app/routes/traffic_data.js

```diff
@@ -4,7 +4,14 @@ var router = express.Router();
 var Sequelize = require('sequelize');
 
 router.get('/', function(req, res) {
-  
+  models.Data.findAll({
+    include: [models.Task]
```

Comment:
I don't think you need this object and include statement either. It should just be `models.Data.findAll()`.